src/Framework/FieldsAPI/DonationAmount.php

```diff
@@ -22,9 +22,9 @@ public static function make($name): DonationAmount
                 Amount::make('amount'),
                 Hidden::make('currency'),
                 Hidden::make('donationType'),
-                Hidden::make('frequency'),
-                Hidden::make('period'),
-                Hidden::make('installments')
+                Hidden::make('subscriptionFrequency'),
+                Hidden::make('subscriptionPeriod'),
+                Hidden::make('subscriptionInstallments')
```

Comment:
Why the prefix?

Reply:
From a data processing perspective, these values (`frequency` `period` `installments`) have no context until you prefix them with `subscription`.  IMO, its more clear what these values represent - as they are specific to creating a subscription, not just a donation.